config-templates/ibm/ess/1.1/storage-class-parameters.json

```diff
@@ -0,0 +1,16 @@
+[
+	{
+		"description": "File system on which directory-based volume should be created",
+		"displayname": "Volume Backend Filesystem",
+		"name": "vol-backend-fs",
+		"required": "true",
+		"type": ""
+	},
+	{
+		"description": "Base path under which all volumes with this storage class will be created. (Must Exist)",
+		"displayname": "Volume Directory Base Path",
+		"name": "vol-dir-base-path",
+		"required": "false",
+		"type": ""
+	}
```

Comment:
Add variable for `name` and `is_default_class` Please refer this https://github.com/IBM/ibm-satellite-storage/blob/develop/config-templates/netapp/netapp-ontap-san/20.07/storage-class-parameters.json

Reply:
Add `name` annotation for `is-default-class`  in storageclass-template ?. e,g 
```
apiVersion: storage.k8s.io/v1
kind: StorageClass
metadata:
name: "{{ name }}"
annotations:
storageclass.kubernetes.io/is-default-class: "{{ is-default-class }}"
```